packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -387,6 +387,20 @@ const conf = (module.exports = convict({
       format: 'url',
     },
   },
+  mxRecordValidation: {
+    enabled: {
+      default: true,
+      doc: 'Feature flag for MX record validation',
+      env: 'FXA_MX_RECORD_VALIDATION',
+      format: Boolean,
+    },
+    exclusions: {
+      default: [],
+      doc: 'List of domains that we will not perform MX record validation on',
```

Comment:
If we were doing release newsletters, it would be good to mention items like this since self hosters etc might wonder how to do this with there own domains.